examples/demo/canvas/canvas.py

```diff
@@ -228,15 +228,13 @@ def clone_plot(clonetool, drop_position):
         "axis",
         "enable_wheel",
         "drag_button",
-        "wheel_zoom_step",
```

Comment:
i thought the plot canvas examples were broken anyway so i dont care much about the changes made to this example.